diff --git a/node_modules/react-native-otp-entry/dist/OtpInput/OtpInput.js b/node_modules/react-native-otp-entry/dist/OtpInput/OtpInput.js
index 0e24a5c..f03c2a1 100644
--- a/node_modules/react-native-otp-entry/dist/OtpInput/OtpInput.js
+++ b/node_modules/react-native-otp-entry/dist/OtpInput/OtpInput.js
@@ -35,7 +35,7 @@ exports.OtpInput = (0, react_1.forwardRef)((props, ref) => {
     const { disabled, numberOfDigits = 6, autoFocus = true, hideStick, focusColor = "#A4D0A4", focusStickBlinkingDuration, secureTextEntry = false, theme = {}, textInputProps, textProps, type = "numeric", } = props;
     const { containerStyle, inputsContainerStyle, pinCodeContainerStyle, pinCodeTextStyle, focusStickStyle, focusedPinCodeContainerStyle, filledPinCodeContainerStyle, disabledPinCodeContainerStyle, placeholderTextStyle, } = theme;
     (0, react_1.useImperativeHandle)(ref, () => ({ clear, focus, setValue: setTextWithRef, blur }));
-    const generatePinCodeContainerStyle = (isFocusedContainer, char) => {
+    const generatePinCodeContainerStyle = (isFocusedContainer, char, isPlaceholderCell) => {
         const stylesArray = [OtpInput_styles_1.styles.codeContainer, pinCodeContainerStyle];
         if (focusColor && isFocusedContainer) {
             stylesArray.push({ borderColor: focusColor });
@@ -43,7 +43,7 @@ exports.OtpInput = (0, react_1.forwardRef)((props, ref) => {
         if (focusedPinCodeContainerStyle && isFocusedContainer) {
             stylesArray.push(focusedPinCodeContainerStyle);
         }
-        if (filledPinCodeContainerStyle && Boolean(char)) {
+        if (filledPinCodeContainerStyle && Boolean(char) && !isPlaceholderCell) {
             stylesArray.push(filledPinCodeContainerStyle);
         }
         if (disabledPinCodeContainerStyle && disabled) {
@@ -64,7 +64,7 @@ exports.OtpInput = (0, react_1.forwardRef)((props, ref) => {
             const isFocusedInput = index === focusedInputIndex && !disabled && Boolean(isFocused);
             const isFilledLastInput = text.length === numberOfDigits && index === text.length - 1;
             const isFocusedContainer = isFocusedInput || (isFilledLastInput && Boolean(isFocused));
-            return (React.createElement(react_native_1.Pressable, { key: `${char}-${index}`, disabled: disabled, onPress: handlePress, style: generatePinCodeContainerStyle(isFocusedContainer, char), testID: "otp-input" }, isFocusedInput && !hideStick ? (React.createElement(VerticalStick_1.VerticalStick, { focusColor: focusColor, style: focusStickStyle, focusStickBlinkingDuration: focusStickBlinkingDuration })) : (React.createElement(react_native_1.Text, { ...textProps, testID: textProps?.testID ? `${textProps.testID}-${index}` : undefined, style: [
+            return (React.createElement(react_native_1.Pressable, { key: `${char}-${index}`, disabled: disabled, onPress: handlePress, style: generatePinCodeContainerStyle(isFocusedContainer, char, isPlaceholderCell), testID: "otp-input" }, isFocusedInput && !hideStick ? (React.createElement(VerticalStick_1.VerticalStick, { focusColor: focusColor, style: focusStickStyle, focusStickBlinkingDuration: focusStickBlinkingDuration })) : (React.createElement(react_native_1.Text, { ...textProps, testID: textProps?.testID ? `${textProps.testID}-${index}` : undefined, style: [
                     OtpInput_styles_1.styles.codeText,
                     pinCodeTextStyle,
                     isPlaceholderCell ? placeholderStyle : {},
